Common/GPU/Vulkan/VulkanContext.cpp

```diff
@@ -795,6 +800,15 @@ VkResult VulkanContext::ReinitSurface() {
 		break;
 	}
 #endif
+#if defined(VK_USE_PLATFORM_DISPLAY_KHR)
+	case WINDOWSYSTEM_DISPLAY:
+	{
+		VkDisplaySurfaceCreateInfoKHR display{ VK_STRUCTURE_TYPE_DISPLAY_SURFACE_CREATE_INFO_KHR };
+		display.flags = 0;
```

Comment:
I was specifically concerned about which flags to pass Retroarch here. It seems that `0` works, though.